.cloudbuild/ci/integration-tests.yaml

```diff
@@ -1,14 +1,9 @@
 timeout: 25m
 options:
   machineType: E2_HIGHCPU_32
+  substitution_option: ALLOW_LOOSE
```

Comment:
I had to lookup what this did. Maybe a brief comment explaining why this is necessary would save future readers the effort.

Reply:
Done